main/pipeline/pipeline.cc

```diff
@@ -268,6 +268,14 @@ vector<ast::ParsedFile> incrementalResolve(core::GlobalState &gs, vector<ast::Pa
                 return what;
             }
         }
+
+#ifndef SORBET_REALMAIN_MIN
+        if (opts.stripePackages) {
+            auto emptyWorkers = WorkerPool::create(0, gs.tracer());
+            what = packager::VisibilityChecker::runIncremental(gs, *emptyWorkers, std::move(what));
+        }
+#endif
```

Comment:
Should we get some timers for these?

Reply:
There are timers in the individual parts, but having one top-level one would be helpful 👍